### PR TITLE
Header & design

### PR DESCRIPTION
**Pull request by ninapeters**
Hi there! I added a Header component to my app. In addition, I have adapted the styling to my design (navigation is planned for later). Please ignore the number of changed files. It looks like a lot more than it actually is.

By adapting the styling, I had to change something on every component. I also refactored the artists and deleted and reordered some files. The tests and snapshots can be ignored, nothing fundamental has changed.

As always, I would appreciate your feedback. ✨

### CODE REVIEW

File: src/components/SavedSongs/SavedSongList.js [hunk 1]
```diff
@@ -12,36 +13,61 @@ SavedSongList.propTypes = {
...
   display: grid;
-  gap: 8px;
+  gap: 32px;
   list-style: none;
   margin: 0;
-  padding: 0;
+  overflow-y: auto;
+  padding: 0 0;
+  scrollbar-width: none;
+  &:before {
```
Comment (JanaHaeusler) [inline src/components/SavedSongs/SavedSongList.js:58]: I have written``::``for these pseudo elements. I think both is correct. ``::`` could be a bit more modern? Found this on [mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/::before)

File: src/components/SavedSongs/SavedSongList.js [hunk 2]
```diff
@@ -12,36 +13,61 @@ SavedSongList.propTypes = {
...
+    content: '';
+    display: block;
+    height: 144px;
+    width: 100%;
+  }
+  &:after {
```
Comment (JanaHaeusler) [inline src/components/SavedSongs/SavedSongList.js:64]: Here as well :)

File: src/styles/GlobalStyle.js [hunk 3]
```diff
@@ -6,17 +6,25 @@ export default createGlobalStyle`
 }
 
 body {
-    font-family: sans-serif;
+    font-family: 'Quicksand', sans-serif;
```
Comment (JanaHaeusler) [inline src/styles/GlobalStyle.js:9]: Very dangerous font-family :D

File: src/styles/GlobalStyle.js [hunk 4]
```diff
@@ -6,17 +6,25 @@ export default createGlobalStyle`
...
+    --primary-light: #679FBE;
+    --primary-regular: #296281;
+    --primary-dark: #3C3C3C;
+    --primary-dark-transparent: #3C3C3CBF;
+    --secondary: #B261A0;
+    --white: #FFF;
+    --white-transparent: #FFFFFFF2;
+    --spotify-green: #08DA73;
+    --spotify-green-transparent: #08DA7333;
+    --cta-red: #FF6262;
```
Comment (JanaHaeusler) [inline src/styles/GlobalStyle.js:25]: Just because I am curious: What is ``cta``?

**Review by JanaHaeusler — COMMENTED** [review]
Awesome work, Nina! Very excited to see your styled app in action. :) Left some ideas and thoughts. Feel free to get back to me, if something is not clear. :) "Happy Nicholas"

File: src/App.js [hunk 5]
```diff
@@ -56,18 +60,22 @@ function App() {
         )}
```
[the remark this answers is not in the document]
Comment (ninapeters) [reply src/App.js:60]: You're right, it's much better that way.

File: src/components/Buttons/Button.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+const ButtonStyled = styled.button`
+  background: var(--white-transparent);
+  border: none;
+  border-radius: ${(props) =>
+    props.isDelteButton ? '34px 10px 10px 34px' : '50%'};
```
[the remark this answers is not in the document]
Comment (ninapeters) [reply src/components/Buttons/Button.js:8]: Thanks for the hint!

File: src/components/SavedSongs/SavedSong.js [hunk 7]
```diff
@@ -33,98 +33,130 @@ export default function SavedSong({
       {toBeDeleted || (
         <ListItem>
           <ButtonWrapper>
-            <ButtonSecondary
+            <PrepareToDeleteButton
```
[the remark this answers is not in the document]
Comment (ninapeters) [reply src/components/SavedSongs/SavedSong.js:36]: I agree, thanks!

File: src/components/SavedSongs/SavedSong.js [hunk 8]
```diff
@@ -33,98 +33,130 @@ export default function SavedSong({
...
     text-align: center;
   }
 `
-const ConfirmationButton = styled(ButtonSecondary)`
-  justify-self: center;
+const ConfirmationButtonWrapper = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+`
+const DelteButton = styled(Button)`
```
[the remark this answers is not in the document]
Comment (ninapeters) [reply src/components/SavedSongs/SavedSong.js:153]: What's my problem with the delete? 😂
Thanks again!

File: src/styles/GlobalStyle.js (hunk 3 above)
Comment (ninapeters) [reply src/styles/GlobalStyle.js:9]: 😎

File: src/styles/GlobalStyle.js (hunk 4 above)
Comment (ninapeters) [reply src/styles/GlobalStyle.js:25]: cta stands for call to action :)

**Pull request merged**